docs/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstoremilvus.md

```diff
@@ -0,0 +1,92 @@
+
+An [example of the connection flow](https://n8n.io/workflows/1960-ask-questions-about-a-pdf-using-ai/) would be: Question and Answer Chain (Retriever connector) -> Vector Store Retriever (Vector Store connector) -> Milvus Vector Store.
+
+### Use the Vector Store Question Answer Tool to answer questions
+
+Another pattern uses the [Vector Store Question Answer Tool](/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolvectorstore.md) to summarize results and answer questions from the Milvus Vector Store node. Rather than connecting the Milvus Vector Store directly as a tool, this pattern uses a tool specifically designed to summarizes data in the vector store.
+
+The [connections flow](https://n8n.io/workflows/2705-chat-with-github-api-documentation-rag-powered-chatbot-with-pinecone-and-openai/) in this case would look like this: AI agent (tools connector) -> Vector Store Question Answer Tool (Vector Store connector) -> Milvus Vector store.
+	
+## Node parameters
+
+--8<-- "_snippets/integrations/builtin/cluster-nodes/vector-store-mode.md"
+
+<!-- vale from-write-good.Weasel = NO -->
+### Get Many parameters
+<!-- vale from-write-good.Weasel = YES -->
+
+* **Milvus Collection**: Select or enter the Milvus Collection to use.
+* **Prompt**: Enter your search query.
+* **Limit**: Enter how many results to retrieve from the vector store. For example, set this to `10` to get the ten best results.
+
+### Insert Documents parameters
+
+* **Milvus Collection**: Select or enter the Milvus Collection to use.
+* **Clear Collection**: Specify whether to clear the collection before inserting new documents.
+
+### Retrieve Documents (As Vector Store for Chain/Tool) parameters
+
+* **Milvus collection**: Select or enter the Milvus Collection to use.
+
+### Retrieve Documents (As Tool for AI Agent) parameters
+
+* **Name**: The name of the vector store.
+* **Description**: Explain to the LLM what this tool does. A good, specific description allows LLMs to produce expected results more often.
+* **Milvus Collection**: Select or enter the Milvus Collection to use.
+* **Limit**: Enter how many results to retrieve from the vector store. For example, set this to `10` to get the ten best results.
+
+## Node options
+
+### Metadata Filter
+
+--8<-- "_snippets/integrations/builtin/cluster-nodes/langchain-root-nodes/vector-store-metadata-filter.md"
+
+### Clear Collection
+
+Available in **Insert Documents** mode. Deletes all data from the collection before inserting the new data.
+
+## Related resources
+
+Refer to [LangChain's Milvus documentation](https://js.langchain.com/docs/integrations/vectorstores/milvus/){:target=_blank .external-link} for more information about the service.
```

Comment:
```suggestion
Refer to [LangChain's Milvus documentation](https://js.langchain.com/docs/integrations/vectorstores/milvus/) for more information about the service.
```

This is definitely used on a lot of pages still, but it's actually deprecated and no longer needed.